manifests/charts/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -115,7 +119,11 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+{{- if not .Values.global.externalIstiod }}
```

Comment:
Not sure what this is for.. can we add comments if we need it?

Reply:
The main difference between the `discovery` and `istiod-remote` charts for the role and clusterrole resources is that the names of the istio-remote ones have `-<istio-namespace>` appended. My thought is that we probably can fix this to use the same name for both (and get rid of lots of these if/else blocks), but I was planning to look at that in a following PR. For this PR, I just want to get rid of the istio-remote charts, but keep everything that is generated EXACTLY the same for now.

Reply:
I am not sure we can just remove the chart without a deprecation window?